packages/cli/tests/e2e/commonUtils.ts

```diff
@@ -94,6 +94,14 @@ export function getSubscriptionId() {
   return cfg.AZURE_SUBSCRIPTION_ID || "";
 }
 
+export function getAzureTenantId() {
+  return cfg.AZURE_TENANT_ID || "";
+}
+
+export function getAzureAccountObjectId() {
+  return cfg.AZURE_ACCOUNT_OBJECT_ID || "c35fa297-dc10-4993-a18c-e8709b8410d1";
```

Comment:
TODO: Will remove this actual value of test account object id and pass the value from github action secret.